### DISCUSSION TITLE
[Blog Plugin] Allow excerpts through page frontmatter

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#5689] [Ideas]
If the post starts with something like an Admonition box will that be shown in the excerpt, which may not look that nice.

It would therefore be a nice feature to have a override in the form of a page frontmatter that allows you to define your own excerpt.

Alternatively have an option that would allow to set the source of the excerpt (i.e. from page or the `description` page frontmatter).

**Comment by squidfunk** [comment]
Thanks for suggesting.

> It would therefore be a nice feature to have a override in the form of a page frontmatter that allows you to define your own excerpt.

This would mean that you would need to implement nested Markdown parsing, in order to define an alternative excerpt. Does any static site generator implement this feature? I'm not sure it's a use case that is of use for many users.

**Reply by Andre601** [reply]
Looking back at this... I assume it would be more difficult to have an option to filter out specific parts (HTML objects) in an excerpt?
I assume it would be kinda overkill for this small thing.

**Reply by squidfunk** [reply]
Definitely possible, but I'm not sure it's worth the effort, given that this discussions got no upvotes or reactions.
